test/services/resolveCourtCase.integration.test.ts

```diff
@@ -532,6 +533,24 @@ describe("resolveCourtCase", () => {
       expectToBeUnresolved(actualCourtCase)
     })
 
+    it("Should return the error if fails to store audit logs", async () => {
+      const error = `Error while calling audit log API`
+      ;(storeAuditLogEvents as jest.Mock).mockImplementationOnce(() => new Error(error))
+
+      let result
+      try {
+        result = await resolveCourtCase(dataSource, courtCases[0], resolution, user)
+      } catch (e) {
+        result = e as Error
+      }
```

Comment:
Nitpick: It could be simplified as `const result = await resolveCourtCase(dataSource, courtCases[0], resolution, user).catch((error) => error as Error)`